### ISSUE TITLE
Fix markuplint thorws SyntaxError when <noscript> followed by CRLF

### ISSUE THREAD

**Issue by takenspc** [markuplint/markuplint#737]
- Markuplint version: `dev`
- Parser lang: HTML
- Node.js version: `18.14.1`
- OS: Windows

## Code Example

```html
<noscript>\r\n
<div>...</div>\r\n
</noscript>
```

## Steps To Reproduce

1. lint html which contains `<noscript>` followed by CRLF

Note: `<noscript>` followed by LF doesn't cause this issue.

## The current behavior

markuplint throws SyntaxError:

```
SyntaxError: Invalid tag syntax: "
<div"

at parseRawTag (packages/@markuplint/html-parser/src/parse-raw-tag.ts:34:9)
at nodeize (packages/@markuplint/html-parser/src/create-tree.ts:199:33)
at createTreeRecursive (packages/@markuplint/html-parser/src/create-tree.ts:64:16)
at nodeize (packages/@markuplint/html-parser/src/create-tree.ts:297:26)
at createTreeRecursive (packages/@markuplint/html-parser/src/create-tree.ts:64:16)
at nodeize (packages/@markuplint/html-parser/src/create-tree.ts:297:26)
at createTreeRecursive (packages/@markuplint/html-parser/src/create-tree.ts:64:16)
at nodeize (packages/@markuplint/html-parser/src/create-tree.ts:297:26)
at createTreeRecursive (packages/@markuplint/html-parser/src/create-tree.ts:64:16)
at createTree (packages/@markuplint/html-parser/src/create-tree.ts:47:9)
at parse (packages/@markuplint/html-parser/src/parse.ts:25:29)
```

## The expected behavior

markuplint doesn't throw SyntaxError.
